Using Same button for Next and Done

Vishalk91-4 opened the pull request:
Fixed and made a single button for both next and done in screen

@llaske could you please review it

llaske commented on v2/js/screens/settings-mysecurity.js:
```diff
@@ -17,7 +17,7 @@ const MySecurity = {
 							<password-box class="security-password" ref="password" @passwordSet="login"/>
 							<icon-button 
-								id="next-btn"
+								id="next-done-btn"
```
Not sure to understand why it need to be changed

llaske commented on v2/js/screens/settings-mysecurity.js:
```diff
@@ -35,7 +35,7 @@ const MySecurity = {
 							<password-box class="security-password" ref="newpassword" @passwordSet="updatePassword"/>
 							<icon-button 
-								id="next-btn"
+								id="next-done-btn"
```
Not sure to understand why it need to be changed

Vishalk91-4 commented:
@llaske 
I changed the id of next-btn and done-btn while merging them, I added it as next-done-btn so next-btn isn't present anywhere
So, I changed it to next-done-btn in those 2 files also

llaske commented:
Nice thanks
